youtube_dl/extractor/prosiebensat1.py

```diff
@@ -349,6 +349,7 @@ class ProSiebenSat1IE(ProSiebenSat1BaseIE):
         r'<p class="att-description">\s*(.+?)\s*</p>',
         r'<p class="video-description" itemprop="description">\s*(.+?)</p>',
         r'<div[^>]+id="veeseoDescription"[^>]*>(.+?)</div>',
+        r'<meta property="og:description" content="(.*?)">',
```

Comment:
@yan12125 Sorry, I don't know what you mean. In the HTML I find:
1. <meta name="description" -> Which is a shortend description
2. <meta property="og:description" - > Which is the one description I used 
3. var contentResources = [{ -> I can use this one too
4. <script type="application/ld+json"> -> I can use this one too
5. <p class="clip-description"  itemprop="description"> -> I can use this one too

So what exactly do you want me to do? 
I don't find _og_search_description() anywhere on the page :-(